flang/lib/Frontend/FrontendActions.cpp

```diff
@@ -264,6 +263,37 @@ static void addDepdendentLibs(mlir::ModuleOp &mlirModule,
   }
 }
 
+// Add to MLIR code target specific items which are dependent on target
+// configuration specified by the user
+static void addTargetSpecificMLIRItems(mlir::ModuleOp &mlirModule,
+                                       CompilerInstance &ci) {
+  const TargetOptions &targetOpts = ci.getInvocation().getTargetOpts();
+  const llvm::Triple triple(targetOpts.triple);
+  if (triple.isAMDGPU()) {
```

Comment:
Please avoid indentation where possible (https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code):

```suggestion
  if (! triple.isAMDGPU())
    return
```

Reply:
done